inst/templates/TopPerformerGraph.r

```diff
@@ -5,8 +5,17 @@ library(pictoralist)
 # Create Pie Chart (Top Performer)
 run <- function(recip, data, spek){
   color_set <- c(PT$DL_BLUE, PT$DL_LIGHT_BORDER, PT$DL_LIGHT_BORDER)
-  percentage <- "90%"
-  goal <- 17/20
+
+  print(spek.measure.comparator.value)
+  recip_data <- filter(data, data$practice == recip)
+  denom_colname <- 'total_scripts'
+  numer_colname <- 'high_dose_scripts'
+  data_denom <- sum(recip_data[denom_colname])
+  data_numer <- sum(recip_data[numer_colname])
```

Comment:
This is clever.  The bracket selection avoids having to deal with dplyr's non-standard evaluation.